src/pages/DetailedLocationPage.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState, useEffect } from 'react';
+import Parse from 'parse/dist/parse.min.js';
+import { useParams } from 'react-router-dom';
+import "../styles/index.css";
+import { Container, Card } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHeart } from '@fortawesome/free-solid-svg-icons';
+import Comment from '../components/common/Comment'
+import Navbar from '../components/common/Navbar'
+import Footer from '../components/common/Footer'
+import CommentFactory from '../components/sections/CommentFactory';
+
+const DetailedLocationPage = () => {
+    const { id } = useParams();
+    const [locationDetails, setLocationDetails] = useState(null);
+
+    const fetchLocationDetails = async () => {
+        const parseQuery = new Parse.Query('Location');
+        parseQuery.include('Country');
+
+        try {
+            const location = await parseQuery.get(id);
+            setLocationDetails(location);
+        } catch (error) {
+            console.error('Error fetching location details:', error);
+        }
+    };
+
+    useEffect(() => {
+        fetchLocationDetails();
+    }, [id]);
+
+
+    return (
+        <div>
+            <Navbar />
```

Comment:
i don't even need to read this code - i am quite sure you are missing a component.

the <Navbar /> is not at the same abstraction level as the spans on line 52!